user_guide_src/source/models/entities.rst

```diff
@@ -190,12 +194,19 @@ current timezone, as set in **app/Config/App.php**:
 .. _entities-property-casting:
 
 Property Casting
-----------------
+================
 
 You can specify that properties in your Entity should be converted to common data types with the ``$casts`` property.
 This option should be an array where the key is the name of the class property, and the value is the data type it
-should be cast to. Casting only affects when values are read. No conversions happen that affect the permanent value in
-either the entity or the database. Properties can be cast to any of the following data types:
+should be cast to.
+
+Property casting affects both read (get) and write (set), but some types affect
```

Comment:
Do we have these specifics documented somewhere?

Reply:
I think it is clear from the following.
https://codeigniter4.github.io/CodeIgniter4/models/entities.html#custom-casting